main.go

```diff
@@ -71,13 +71,13 @@ func main() {
 	// to more cleanly manage it
 
 	// Build router from middleware
-	var router = router.NewTastyBoiRouter(userMiddleware, recipeMiddleware, ingredientMiddleware, serverMiddleware, householdMiddleware)
+	var tastyRouter = router.NewTastyBoiRouter(userMiddleware, recipeMiddleware, ingredientMiddleware, serverMiddleware, householdMiddleware)
```

Comment:
lmao wat.

Reply:
var router actually conflicts with the package name so I renamed it not. I already used `r` below so this was the the first thing I thought of.

Reply:
mmmmmm.... router